### DISCUSSION TITLE
Best practise for user's details implementation?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10759] [Archive]
Hi,
I'd like to add properties to users : date/place of birth, jobs, etc.
Is it better to
1. add fields to the directus_users collection (using Directus admin)
2. or to create an "user" collection (related with the directus_users collection by O2O aka M2O/max=1)

Nr. 1 is straightforward, but is it safe towards  Directus future versions?
Nr. 2 is neat but with more homework through out the app.

**Comment by rijkvanzanten** [comment]
It's totally up to you!

You're two comments at the bottom are correct. Number 1 would be more straightforward, but you have the chance it might conflict with system fields added in the future (though realistically, we haven't really changed the schema for directus_users much since the start of v9).
